docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -179,6 +179,13 @@ as root:
   kubeadm join --token <token> <master-ip>:<master-port> --discovery-token-ca-cert-hash sha256:<hash>
 ```
 
+To let kubectl work in the following steps, you need to run these commands as a regular user (which is part of the output above):
+```
+  mkdir -p $HOME/.kube
+  sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+  sudo chown $(id -u):$(id -g) $HOME/.kube/config
```

Comment:
as @mikedanese pointed out, better add one line:
```
sudo chmod 600 $HOME/.kube/config
```
Then it lgtm.

Reply:
@tengqm I don't think this should change. What's there now is good as-is